### PR TITLE
Require at least one space on the beginning of comments

### PR DESCRIPTION
**Pull request by marcogbarcellos**
Add rule to require at least one space on comments.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-opencollective",
-  "version": "1.4.0",
```
Comment (znarf) [inline package.json]: Don't be shy, 1.5.0 :-)

File: index.js [hunk 2]
```diff
@@ -59,6 +59,10 @@ module.exports = {
     // https://eslint.org/docs/rules/no-var
     'no-var': 2,
 
+    // require at least one whitespace after comments( // and /*)
+    // https://eslint.org/docs/rules/spaced-comment
+    'spaced-comment': ["error", "always"],
```
Comment (znarf) [inline index.js]: In this file, we also follow our eslint rules, so simple quotes.

For that kind of rules, I usually start with a simple 'warn' to make sure we don't break CI in other projects.

Comment (marcogbarcellos) [reply index.js]: i see, makes sense. will update.

**Comment by marcogbarcellos** [comment]
can we merge this one now @znarf ?

File: index.js [hunk 3]
```diff
@@ -75,6 +75,9 @@ module.exports = {
     // require trailing commas
     // https://eslint.org/docs/rules/comma-dangle
     'comma-dangle': [1, 'always-multiline'],
+    // require at least one whitespace after comments( // and /*)
+    // https://eslint.org/docs/rules/spaced-comment
+    'spaced-comment': [1, "always"],
```
Comment (znarf) [inline index.js]: Sorry, still double quotes here. `npm run lint -- --fix`

**Comment by znarf** [comment]
Yes, just fix the small quotes issue and we can merge it.

File: index.js (hunk 3 above)
Comment (marcogbarcellos) [reply index.js]: omg you're right. sorry.

**Pull request merged**